bin/update_hostname_validator.php

```diff
@@ -0,0 +1,130 @@
+<?php
```

Comment:
I invoke it with PHP: `php bin/update_hostname_validator.php`; this has been how it's always been invoked since introduced to ZF2, and is only meant for updating the TLD list.